blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/HttpMetricsIndexHandler.java

```diff
@@ -24,7 +23,7 @@
     private DiscoveryIO discoveryHandle;
 
     public HttpMetricsIndexHandler() {
-        loadDiscoveryModule();
+        discoveryHandle = QueryDiscoveryModuleLoader.getDiscoveryInstance();
```

Comment:
@chinmay-gupte I wasnt aware of visibleForTesting. It still feels hacky, but at this point lets just proceed with it.